frontend/packages/client/src/contexts/NotificationService.js

```diff
@@ -64,74 +81,68 @@ const NotificationServiceProvider = ({ children }) => {
   const getUserSettings = async () => {
     try {
       //here we call api
-      setNotificationSettings();
-    } catch {
-      throw new Error('cannot get user settings');
-    }
-  };
-
-  const unsubscribeCommunity = async (communityId) => {
-    try {
-      //here we call api
+      const { communitySubscription, isSubscribedFromCommunityUpdates } =
+        INIT_NOTIFICATION_SETTINGS;
```

Comment:
This is to fill some mock data for the setting the state on line 86, once api methods are implemented will be replaced with the response of the api call